Source/WebCore/platform/graphics/skia/FontCascadeSkia.cpp

```diff
@@ -54,6 +54,8 @@ void FontCascade::drawGlyphs(GraphicsContext& graphicsContext, const Font& font,
     SkPaint paint = static_cast<GraphicsContextSkia*>(&graphicsContext)->createFillPaint();
     paint.setAntiAlias(font.allowsAntialiasing());
     paint.setImageFilter(static_cast<GraphicsContextSkia*>(&graphicsContext)->createDropShadowFilterIfNeeded(GraphicsContextSkia::ShadowStyle::Outset));
+    static_cast<GraphicsContextSkia*>(&graphicsContext)->setFillSource(paint);
```

Comment:
I'm not sure about this, I think the fill color is what we always want for text, in that case we can make the helper private.